extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesPeonLifecycle.java

```diff
@@ -288,7 +288,11 @@ private TaskStatus getTaskStatus(long duration)
             TaskStatus.class
         );
       } else {
-        taskStatus = TaskStatus.failure(taskId.getOriginalTaskId(), "task status not found");
+        log.info(
```

Comment:
```suggestion
        log.warn(
```

Why info and not warn? This seems like it's not normal operations